web-jehlomat/src/Components/Buttons/TextButton/TextButton.tsx

```diff
@@ -5,24 +5,36 @@ import Typography from '@mui/material/Typography/Typography';
 
 interface ITextButton extends ButtonHTMLAttributes<HTMLButtonElement> {
     text: String;
+    fontSize?: string;
+    fontWeight?: string;
+    color?: string;
+    underline?: boolean;
 }
 
 const Button = styled.button`
     border: none;
     box-shadow: none;
     background: transparent;
-    color: ${white};
     cursor: pointer;
-    font-size: 16px;
     line-height: 19px;
     text-align: center;
     margin-bottom: 10px;
 `;
 
-const TextButton: FC<ITextButton> = ({ text, ...props }) => {
+const TextButton: FC<ITextButton> = ({ text, fontSize, fontWeight, color, underline, ...props }) => {
```

Comment:
Mate niekto nazor na to, aku logiku zvolit, aby to bolo konzistentne zo zvyskom?
Teraz menim TextButton componentu cez props.
Vs.
Nova komponenta s danymi stylmi, akurat sa obavam, ze potom budeme mat nekonecno variant (rozne colors, fontsizes, fontweights).
Dik

Reply:
yeah, it could be a problem. You could introduce "type" of a text button.
Like: primary/secondary, small / medium / big
If it still has a lot of options, I would introduce 1 more local component for such case.